backend/apps/accounts/apps.py

```
@@ -7,3 +7,16 @@ class AccountsConfig(AppConfig):
+        try:
+            from django_q.models import Schedule
+
+            Schedule.objects.get_or_create(
+                name="purge-expired-sessions-daily",
+                defaults={
+                    "func": "apps.accounts.tasks.purge_expired_sessions_task",
+                    "schedule_type": Schedule.DAILY,
+                },
+            )
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)